store/metrics.go

```diff
@@ -34,7 +36,14 @@ func newMetrics() (m *metrics, err error) {
 	if err != nil {
 		return nil, err
 	}
-	m.headHeightReg, err = meter.RegisterCallback(m.observeHeight, m.headHeightInst)
+	m.tailHeightInst, err = meter.Int64ObservableGauge(
+		"tail_store_head_height_gauge",
```

Comment:
```suggestion
		"hdr_store_tail_height_gauge",
```

match to head height

Reply:
[8d6d875](https://github.com/celestiaorg/go-header/pull/283/commits/8d6d875c3f3ac193bf7fee4c7f1d5b311549502d)